src/licensemanager2/workload_managers/slurm/cmd_utils.py

```diff
@@ -294,3 +296,61 @@ async def sacctmgr_modify_resource(
         logger.warning(modify_resource_stdout)
         return False
     return True
+
+
+def return_formated_squeue_out() -> str:
+    """Call squeue via Popen and return the output.
+
+    return type: str
+    """
+
+    try:
+        stdout, _ = subprocess.Popen(
+            [SQUEUE_PATH, "--noheader", "--format='%A|%M|%T'"]
+        )
+    except subprocess.CalledProcessError as e:
+        logger.error(e)
+        raise e
+
+    return stdout.decode().strip()
+
+
+def _total_time_in_seconds(time_string: str) -> int:
+    """Return the sum of the hours minutes and seconds (in seconds)."""
+    MINUTE = 60
+    HOUR = 60 * MINUTE
+    DAY = 24 * HOUR
+
+    days = 0
+    hours = 0
+    if "-" in time_string:
+        days, time_string = time_string.split("-")
+
+    measure_of_time = time_string.count(":")
+
+    if measure_of_time == 2:
+        hours, minutes, seconds = time_string.split(":")
+    else:
+        minutes, seconds = time_string.split(":")
+
+    return (
+        int(days) * DAY + int(hours) * HOUR + int(minutes) * MINUTE + int(seconds)
+    )
```

Comment:
Fix mypy and also a more clean way of parsing, since we are already importing `re`
```suggestion
    days = 0
    hours = 0
    splitted_time = [int(value) for value in re.split("-|:", time_string)]
    splitted_time_len = len(splitted_time)

    if splitted_time_len == 3:
        days, hours, minutes, seconds = splitted_time
    elif splitted_time_len == 2:
        hours, minutes, seconds = splitted_time
    else:
        minutes, seconds = splitted_time

    return days * DAY + hours * HOUR + minutes * MINUTE + seconds
```